assets/css/main.scss

```
@@ -1563,3 +1507,5 @@ section.sponsors .grid {
   border: none;
   margin: 35px 0;
 }
+
+@import 'footer';
```

Add newline at end of file:

```suggestion
@import 'footer';

```